File: Uri/1046.py
x = raw_input().split(' ')
i = int(x[0])
f = int(x[1])

if i > f:
    t = 24-i+f
elif f > i:
    t = f-i
else:
    t = 24

print "O JOGO DUROU %d HORA(S)"%t

File: Uri/1046.c
#include<stdio.h>
#include<stdlib.h>

int main(){

    int x, y;
    scanf("%d %d", &x, &y);

    if (y > x)
        printf("O JOGO DUROU %d HORA(S)\n", y - x);
    else
        printf("O JOGO DUROU %d HORA(S)\n", 24 - x + y);
}
